src/plugins/orangehrmRecruitmentPlugin/entity/Decorator/CandidateVacancyDecorator.php

```diff
@@ -64,7 +64,7 @@ public function getCandidateVacancyStatus(): array
     {
         $candidateVacancyStatus = $this->candidateVacancy->getStatus();
         return [
-            'id' => array_flip(CandidateAPI::STATUS_MAP)[$candidateVacancyStatus],
+            'id' => array_flip(CandidateService::STATUS_MAP)[$candidateVacancyStatus],
```

Comment:
same here. but if you are reusing this method multiple place, keep it

Reply:
yeah reusing